src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -698,35 +730,66 @@ sub-<label>/[ses-<label>/]
         sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_epi.json
 ```
 
-The phase-encoding polarity (PEpolar) technique combines two or more Spin Echo
-EPI scans with different phase encoding directions to estimate the underlying
-inhomogeneity/deformation map. Examples of tools using this kind of images are
-FSL TOPUP, AFNI 3dqwarp and SPM. In such a case, the phase encoding direction is
-specified in the corresponding JSON file as one of: `i`, `j`, `k`, `i-`, `j-`,
-`k-`. For these differentially phase encoded sequences, one also needs to
-specify the Total Readout Time defined as the time (in seconds) from the center
-of the first echo to the center of the last echo (aka "FSL definition" - see
-[here](https://fsl.fmrib.ox.ac.uk/fsl/fslwiki/topup/Faq#How_do_I_know_what_phase-encode_vectors_to_put_into_my_--datain_text_file.3F) and
-[here](https://lcni.uoregon.edu/kb-articles/kb-0003) how to calculate it). For
-example
+The [`dir-<label>`](../99-appendices/09-entities.md#dir) key/value pair is
+distinguish between different phase-encoding directions is REQUIRED for these
+files, and SHOULD NOT be used to replace the required `PhaseEncodingDirection`
+(nor any other acquisition parameter, as prescribed by the
+[common principles](../02-common-principles.md#file-name-structure).)
+
+Required fields:
+
+| **Key name**           | **Requirement level** | **Data type** | **Description**                                                                    |
+| ---------------------- | --------------------- | ------------- | ---------------------------------------------------------------------------------- |
+| PhaseEncodingDirection | REQUIRED              | [string][]    | See [in-plane spatial encoding](#in-plane-spatial-encoding) table of fields.       |
+| TotalReadoutTime       | REQUIRED              | [number][]    | See [in-plane spatial encoding](#in-plane-spatial-encoding) table of fields.       |
+
+For example:
 
 ```JSON
 {
    "PhaseEncodingDirection": "j-",
-   "TotalReadoutTime": 0.095,
-   "IntendedFor": "func/sub-01_task-motor_bold.nii.gz"
+   "TotalReadoutTime": 0.095
+}
+```
+
+As for other EPI sequences, these field mapping sequences may have any of the
+[in-plane spatial encoding](#in-plane-spatial-encoding) metadata keys.
+However, please note that `PhaseEncodingDirection` and `TotalReadoutTime` keys
+are REQUIRED for these field mapping sequences.
+
+### Expressing the MR protocol intent for fieldmaps
+
+#### Using `B0FieldIdentifier` metadata
+
+The general purpose `B0FieldIdentifier` MRI metadata is RECOMMENDED for the
+prescription of the B<sub>0</sub> field estimation intent of the original acquisition
+protocol.
+This encoding fully overrides the `IntendedFor` approach (see below).
+However, the `IntendedFor` encoding is permitted and RECOMMENDED in combination
+with the `B0FieldIdentifier` to maintain compatibility with outdated software.
```

Comment:
I think of "override" as more like "replaces", but we're recommending duplication. What about:

```suggestion
`B0FieldIdentifier` and `B0FieldSource` duplicate the capabilities of
the original `IntendedFor` approach while permitting more complex use cases.
It is RECOMMENDED to use both approaches to maintain compatibility with
tools that support older datasets.
```